while_basic_8 (1).py

```diff
@@ -0,0 +1,75 @@
+def countTo_while(max_number):
+    number = 0
+    while number <= max_number:
+        print(number)
+        number += 1
+
+def countTo_for(max_number):
+    for number in range(max_number + 1):
+        print(number)
+
+def feedList_2():
+    names = []
+    user_input = input("Please write name: ")
+    while user_input != "stop":
+        names.append(user_input)
+        user_input = input("Please write name: ")
+    print(names)
+
+def iterateList_while(input_list):
+    index = 0
+    while index < len(input_list):
+        print(input_list[index])
+        index += 1
+
+def count_toMax(max_number):
+    number = 0
+    while number <= max_number:
+        print(number)
+        number += 1 
+
+def feedList_1():
+    food = []
+    while True:
+        user_input = input("Enter food you like: ")
+        food.append(user_input)
+        if user_input == "stop":
+            return food
+
+def feedList():
+    food = []
+    user_input = input("Enter food you like: ")

```

Comment:
במצבים כאלו שהסטרינג כתוב גם בשורה 41 וגם בשורה 44, הייתי שומר אותו במשתנה
question = "Enter food you like: "
כדי שלא יהיה שכפול קוד